### PR TITLE
remove MLNX/BRCM specific initialization code

### PR DESCRIPTION
**Pull request by lguohan**

### CODE REVIEW

File: syncd/syncd.cpp [hunk 1]
```diff
@@ -1167,7 +1161,7 @@ int main(int argc, char **argv)
 
     initialize_common_api_pointers();
 
-    sai_status_t status = sai_switch_api->initialize_switch(0, "0xb850", "", &switch_notifications);
+    sai_status_t status = sai_switch_api->initialize_switch(0, "", "", &switch_notifications);
```
Comment (kcudnik) [inline syncd/syncd.cpp]: is this value not needed for bcm ?

**Comment by kcudnik** [comment]
Where then will be specific code executed ?

**Comment by stcheng** [comment]
I think an option -p will be added to provide the profile.

**Comment by stcheng** [comment]
ACK

File: syncd/syncd.cpp (hunk 1 above)
Comment (lguohan) [inline syncd/syncd.cpp]: Not needed.

**Pull request merged**